imap_processing/mag/l0/decom_mag.py

```diff
@@ -1,15 +1,21 @@
-"""Decom MAG data packets using MAG packet definition."""
-
+"""Methods for processing raw MAG packets into CDF files for level 0 and level 1a."""
+import dataclasses
 import logging
+from collections import defaultdict
 from pathlib import Path
 
+import numpy as np
+import xarray as xr
 from bitstring import ReadError
 from space_packet_parser import parser, xtcedef
 
 from imap_processing import imap_module_directory
 from imap_processing.ccsds.ccsds_data import CcsdsData
+from imap_processing.cdf.global_attrs import ConstantCoordinates
+from imap_processing.mag import mag_cdf_attrs
 from imap_processing.mag.l0.mag_l0_data import MagL0, Mode
 
+logging.basicConfig(level=logging.ERROR)
 logger = logging.getLogger(__name__)
```

Comment:
Why don't we want warning messages? I'd just leave this alone, but we should only update this logger and not the global level.
```suggestion
logger = logging.getLogger(__name__)
logging.setLevel(logging.ERROR)
```

Reply:
Oh yeah, this is left over from something else. Let me take it out.